recipes/stereo-pipeline/build.sh

```diff
@@ -0,0 +1,193 @@
+#!/bin/bash
+
+# Build dependencies first
+
+# Multiview
+cd $SRC_DIR/MultiView
+mkdir -p build && cd build
+cmake ..                                                              \
+    -DCMAKE_BUILD_TYPE=Release                                        \
+    -DMULTIVIEW_DEPS_DIR=${PREFIX}                                    \
+    -DCMAKE_OSX_DEPLOYMENT_TARGET=10.13                               \
```

Comment:
This should be conditionalized on whether or not we're OSX, not set for every `unix` build.

Reply:
Adressed.